docker-compose.yaml

```diff
@@ -0,0 +1,33 @@
+services:
+  cinema_service:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8001:8000"
+    volumes:
+      - ./:/app
+      - my_media:/files/media
+    command: >
+      sh -c "python manage.py wait_for_db &&
+      python manage.py makemigrations &&
+      python manage.py migrate &&
+      python manage.py runserver 0.0.0.0:8000"

```

Comment:
Ensure that the `wait_for_db` script is implemented correctly to handle database readiness before running migrations and starting the server. This is crucial for avoiding race conditions during startup.